buddyboss/buddyboss-platform
# 781 - Fixed the forum auto suggest on forum archive page

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #781 .

### How to test the changes in this Pull Request:

1. Enable the forum banner and search from theme settings >> forums (If not enabled).
2. Logged as a normal user and search for forums or discussions.
3. See the auto suggest results

### Proof Screenshots or Video
https://www.loom.com/share/7ddd640f37b542ebad8f842490049d87



### Other information:




### Changelog entry

> Fixed the forum auto suggest on forum archive page
